tpu_inference/layers/vllm/fused_moe.py

```diff
@@ -162,7 +161,10 @@ def _gmm_all_reduce(lhs, rhs, rhs_scale, rhs_bias, group_sizes):
         m, g, n, k = lhs.shape[0], *rhs.shape
         tm, tk, tn = _get_tiling_size_for_gmm_kernel(m, k, n, g)
         if rhs_bias is not None:
-            shard_id = jax.lax.axis_index("model")
+            model_shard_id = jax.lax.axis_index("model")
```

Comment:
> if "attn_dp" in mesh.axis_names

this looks too messy.

Given that `jax.lax.axis_index` can take in tuple as an argument, wouldn't it be possible to do something like this? `jax.lax.axis_index(ShardingAxisName.MLP_TENSOR).sum()`?